src/SystemCommands-VariableCommands/SycAbstractAllInstVarAccessorsCommand.class.st

```diff
@@ -1,66 +0,0 @@
-"
-I command to launch the Abstract refactorings: create accessors and abstract the direct instance variable uses into accessors. 
```

Comment:
Was this class removal wanted?

Reply:
I don't understand this change. At some point, Iceberg did some unwished changes to my working image. I tried to revert it. Was it a left over, I can't tell for sure.